benchmarks/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+(cd benchmarks/fixtures && npm ci);
```

Comment:
Would `npm ci --prefix benchmarks/fixtures` work?

Reply:
Absolutely, thanks! Done in https://github.com/netlify/zip-it-and-ship-it/pull/411/commits/91b08cf7dd9b50186d4c410a99bfdf6fc2cec6e9.